Sources/SourceControl/GitRepository.swift

```diff
@@ -15,8 +15,8 @@ import func POSIX.getenv
 import enum POSIX.Error
 import class Foundation.ProcessInfo
 
-enum GitRepositoryProviderError: Swift.Error {
-    case gitCloneFailure(url: String, path: String)
+enum GitRepositoryProviderError: ErrorProtocol {
```

Comment:
Ah right, I missed converting that one back again.  Did I mention this PR is a work in progress...?  :)
Thanks for the heads-up.